### DISCUSSION TITLE
Github markdown doesn't

### DISCUSSION THREAD

**Discussion by anterpin** [community/community#55687] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

Github markdown doesn't parse some mathematical expression as intended.
In the docs, Github says that it uses Mathjax as a Markdown parser, but I went on the Mathjax site and it parses the following content good, on the contrary of Github.

**Example text**

Мы используем гравитационный потенциал для иллюстрации основной формы вычислений в моделировании всех пар N-тел. В следующих вычислениях мы используем жирный шрифт для обозначения векторов (обычно в 3D). Учитывая $N$ тел с начальным положением $\textbf{x}_i$ и скоростью $\textbf{v}_i$ для $ \leqslant i \leqslant N$, вектор силы $ \textbf{f}_{ij} $на теле $i$, вызванной его гравитационным притяжением к телу $j$, задается следующим образом:

$\{m_i\}$

$$
\textbf{f}_{ij} = G\frac{m_im_j}{||\textbf{r}||^2}\cdot\frac{\textbf{r}_{ij}}{||\textbf{r}_{ij}||}
$$

```math
\textbf{f}_{ij} = G\frac{m_im_j}{||\textbf{r}||^2}\cdot\frac{\textbf{r}_{ij}}{||\textbf{r}_{ij}||}
```

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @anterpin ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by dpvc** [comment]
See #36915 for the description of what is going on here, and how you can work around it.
